extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchBufferAggregator.java

```diff
@@ -30,17 +30,16 @@
 
 import java.nio.ByteBuffer;
 import java.util.HashMap;
+import java.util.IdentityHashMap;
 import java.util.Map;
 
 public class SketchBufferAggregator implements BufferAggregator
 {
   private final ObjectColumnSelector selector;
   private final int size;
   private final int maxIntermediateSize;
-
-  private NativeMemory nm;
-
   private final Map<Integer, Union> unions = new HashMap<>(); //position in BB -> Union Object
```

Comment:
As I mentioned earlier, this field was already there, and this pr is not to refactor `SketchBufferAggregator` but fixing a bug exists in druid 10. It will be nice if you can provide some good reasons why it has to be changed now.